openhcl/openhcl_attestation_protocol/src/igvm_attest/get.rs

```diff
@@ -345,6 +345,8 @@ pub mod runtime_claims {
         pub tpm_enabled: bool,
         /// Whether the TPM states is persisted
         pub tpm_persisted: bool,
+        /// Whether Azure-specific devices are allowed through the device filter
+        pub azure_filtered_devices_allowed: bool,
```

Comment:
maybe 'filtered_vpci_devices_allowed'.  

I think we can drop the 'azure_' prefix and Azure comment reference above; Azure paravisor will filter to Azure devices, but it's not part of the attested config architecture